doc/gov/chapters/chapter02.md

```diff
@@ -13,7 +13,7 @@
 <a name="2.1"></a>
 ## 2.1 Introduction
 
-As an open-source community organization, CNTT has developed the following process to ensure delivery on CNTT objectives. This process has several goals. First, inform new participants of the resources available upon joining the community. Second, communicate the organizational and legal structures upon which the community operates. Help new members become fully engage & contributing effectively to the work streams.
+As an open-source community organization, CNTT has developed the following process to ensure delivery on CNTT objectives. This process has several goals. First, inform new participants with the background and overview of this open-source initiative. Second, communicate the organizational and legal structures upon which the community operates. And third, help new members become fully engage & contributing effectively to the goals of CNTT via its various work streams.
```

Comment:
I still believe, that CNTT does not have membership, therefore it can not have members. What about new contributors or participants again?

Reply:
CNTT participants join mailing lists - which is our proxy for membership in CNTT - I can put in the mailing list that participation == agreement to the Term of Reference?

Reply:
@CsatariGergely. I meant to take out all the 'member' references. So I'll remove that in the last sentence. Good catch.

Reply:
> CNTT participants join mailing lists - which is our proxy for membership in CNTT - I can put in the mailing list that participation == agreement to the Term of Reference?

+1

Reply:
@rabi-abdel @mf4716 @bfcohen @mtnskiier @phrobb @walterkozlowski @josoinin @toshiwkayamaKDDI @ztehujie @william-diego @Lingli-Deng-cmcc @jisu-park @jiaqiang-cmcc @rt063356 please review.